packages/core/content-manager/admin/src/preview/pages/Preview.tsx

```diff
@@ -211,11 +211,9 @@ const ProtectedPreviewPageImpl = () => {
           permission.action.includes('explorer.read')
         )}
       >
-        {({ permissions }) => (
-          <DocumentRBAC permissions={permissions}>
-            <PreviewPage />
-          </DocumentRBAC>
-        )}
+        <DocumentRBAC permissions={permissions}>
```

Comment:
I'm cool with this idea, but a bit bothered that the permissions are not handled the same here vs the EditViewPage. What are your thoughts on that. I'm cool to move forward as is. Just wondering if we should leave a comment to update the EditViewPage, update the EditViewPage now, or decide that it's fine if they handle it differently.

Reply:
tbh i didn't check the edit view page implementation, i did what made sense to me here. i'm gonna look into how that one works now

Reply:
update: there's actually an issue with Page.Protect that the edit view page relies on, I'll bring this up at the frontend sync